lntest/itest/macaroons.go

```diff
@@ -119,6 +120,45 @@ func testMacaroonAuthentication(net *lntest.NetworkHarness, t *harnessTest) {
 	res, err := client.NewAddress(ctxt, newAddrReq)
 	require.NoError(t.t, err, "get new address")
 	assert.Contains(t.t, res.Address, "bcrt1", "check regtest address")
+
+	// Seventh test: Bake a macaroon that can only access exactly two RPCs
```

Comment:
Perhaps we should break these up into actual sub-tests using `t.Run`? This would also allow them to potentially all be ran in parallel.